Improvement/adds item metadata with assembly options

lucasayb opened the pull request:
#### What is the purpose of this pull request?
Adds item metadata with assembly options
<!--- Describe your changes in detail. -->

#### What problem is this solving?
Missing `itemMetadata` section
<!--- What is the motivation and context for this change? -->

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

jgfidelis commented:
What information do you really need?

All of these fields?
What are you trying to achieve?

lucasayb commented:
@jgfidelis I've added those fields to integrate the store where I'm working at with the `assemblyOptions`. But I've reviewed the pull request and removed some unnecessary fields.
